autoRSA.py

```diff
@@ -572,6 +577,11 @@ async def rsaadd(ctx, broker=None, credentials=None):
                         raise Exception(
                             "Invalid credentials. Just enter your email for Fennel."
                         )
+                elif broker =="Dspac":
+                    if len(credentials.split(":")) != 1:
+                        raise Exception(
+                            "Invalid credentials. Use this format for Dspac: email:password"

```

Comment:
this message still has "email" please change to username

Reply:
updated